test/batch/batchCreate_test.go

```diff
@@ -158,6 +223,83 @@ func TestBatchCreate_integration(t *testing.T) {
 		testsuit.CleanUpWeaviate(t, client)
 	})
 
+	t.Run("POST /batch/references?consistency_level={level}", func(t *testing.T) {
```

Comment:
Could you add a short description what you're testing here? one-liner is enough
Same above

Reply:
done